mlir/include/mlir/Dialect/Polynomial/IR/Polynomial.td

```diff
@@ -123,9 +123,18 @@ def Polynomial_RingAttr : Polynomial_Attr<"Ring", "ring"> {
   let parameters = (ins
     "Type": $coefficientType,
     OptionalParameter<"IntegerAttr">: $coefficientModulus,
-    OptionalParameter<"PolynomialAttr">: $polynomialModulus
+    OptionalParameter<"PolynomialAttr">: $polynomialModulus,
+    OptionalParameter<"IntegerAttr">: $primitiveRoot
   );
 
+  let builders = [
+    AttrBuilder<
+        (ins "::mlir::Type":$coefficientTy,
+             "IntegerAttr":$coefficientModulusAttr,
+             "PolynomialAttr":$polynomialModulusAttr), [{
```

Comment:
Nit: let's me consistent about using the `::mlir::` prefix for types.

Reply:
Fixed in 2d0fad3b945